test/org/zaproxy/zap/spider/parser/SpiderHtmlFormParserUnitTest.java

```diff
@@ -939,11 +942,12 @@ public void shouldSetValuesToFieldsWithNoValueWhenParsingGetForm() {
     @Test
     public void shouldSetValuesToFieldsWithNoValueWhenParsingPostForm() {
         // Given
-        SpiderHtmlFormParser htmlParser = createSpiderHtmlFormParser();
+        DefaultValueGenerator valueGenerator = new DefaultValueGenerator();
+    	SpiderHtmlFormParser htmlParser = createSpiderHtmlFormParser(valueGenerator);
```

Comment:
It's possible. We do have plans to standardize and go through an auto-formatting activity but it's on the to-do list.... 

If you need to adjust something to be consistent I suggest just copying the indentation on one of the surrounding lines and pasting it, not optimal but workable.